linetools/analysis/interactive_plot.py

```diff
@@ -86,10 +86,14 @@ def on_keypress_navigate(self, event):
         """
         # Requiring inaxes for all of these now
         if (event.key in self.nav_dict['nav']) and event.inaxes:
-            ltgu.navigate(self.nav_dict, event, flux=self.fl, wave=self.wa)
-            self.ax.set_xlim(self.nav_dict['x_minmax'])
-            self.ax.set_ylim(self.nav_dict['y_minmax'])
-            self.fig.canvas.draw()
+            try:
+                ltgu.navigate(self.nav_dict, event, flux=self.fl, wave=self.wa)
+            except KeyError:
+                pass
+            else:
```

Comment:
What is this `else` doing?

Reply:
just continuing the code if we don't hit the error.